docker-compose.yaml

```diff
@@ -0,0 +1,92 @@
+version: "3"
+
+volumes:
+  db_data: {}
+  nginx: {}
+  cbv3_django_prototype: {}
```

Comment:
@billglover Just wanted to point out that we have all these named volumes defined but we're not really using these. Changing `./db_data` to just `db_data` on line 19, `./nginx` to `nginx` on line 55 and so on will make sure we use named volumnes instead of bind mounts. This should fix our problems on Windows too.

Reply:
@sunu Are you saying that if @nuageklow makes those changes in the `docker-compose.yaml` file locally, it would probably work on her machine?

Reply:
@lpatmo i actually posted the same solution lastnight earlier on this same thread..i guess i worded it in a different way..? 
https://github.com/codebuddies/django-concept/pull/44#issuecomment-576696357

but yes it worked and everything was installed properly.

for `docker-compose run --rm manage shell` command it didnt work. i'll find a way and post it if i have a solution.

Reply:
@lpatmo Yes, that's my assumption. It seems they have already figured out as much in https://github.com/codebuddies/django-concept/pull/44#issuecomment-576696357

Reply:
Oh sorry, I completely missed your comment from yesterday! Too many posts 😅 

So the remaining items to do are:
1/ Update the docker file in this PR
2/ Figure out the shell command
3/ sunu's `makefile` idea in a separate PR (maybe)?

Reply:
Slight nuance... making the change and removing the `./` to use volume mounts means volumes cannot be mounted from within the project folder and so code doesn't get mapped through to the running application. This means development on the host isn't reflected in the running application so won't work for us.

I'm looking through documentation now to see if we have to drop back to bind mounts to achieve the set-up we are working towards.

Reply:
@sunu @billglover do you think it would be something windows specific only? i was sort of worried abt things might not be placed properly even thought it seemed like it ran smoothly. 

also any expected end result be included would be ideal (e.g. how to define the volume is installed properly) sorry i guess im too new on this..